Source/Library/UIColor+Hex.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+public extension UIColor {
+    
+    convenience init(hex string: String) {
```

Comment:
I think that duplicating the code it's a bad idea, for the reason that, if we have to go to Swift 4 for instance, or we have to do something about it, we are going to have to remember that Lightbox is breaking because it didn't update Hue properly.

I agree that having it as a dependency looks also "bad", but then I wouldn't even use hex codes if we don't want to use it as one. At the end of the day, we don't use it in that many cases. I've counted 5 cases and 3 of them are the same color.